tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/executors/executorservodriver.py

```diff
@@ -23,8 +26,62 @@ def do_delayed_imports():
     global webdriver
     import webdriver
 
+    global ServoCommandExtensions
+    class ServoCommandExtensions(object):
```

Comment:
Why is this inside `do_delayed_imports`?

Reply:
Because it relies on `webdriver.client.command`, and `webdriver` is delayed.